keyboards/checkerboards/g_idb60/readme.md

```diff
@@ -0,0 +1,13 @@
+# G-IDB60 
+
+60% drop-in PCB for IDB60 feauturing Tsangan bottom row, split right shift, and split backspace. 
+
+- Keyboard Maintainer: [Nasp](https://github.com/npspears)
+- Hardware Supported: G-IDB60 PCB
+- Hardware Availability: https://www.checkerboards.xyz/gidb60.html
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader here? 


Eg, is there a hardware reset button, etc.

Reply:
Done